app/routes.py

```diff
@@ -58,7 +59,7 @@ async def registries_all():
     "/registries/",
     response_model=schemas.RegistryResponse,
     responses={
-        400: {
+        HTTPStatus.BAD_REQUEST: {
```

Comment:
fyi since statuses are `IntEnum`, it should be fine (so no need to use `HTTPStatus.BAD_REQUEST.value` i.e. `int` explicitly)